src/modules/logger/logger.cpp

```diff
@@ -730,11 +743,27 @@ void Logger::run()
 			param_get(_sdlog_mode_handle, &_sdlog_mode);
 		}
 
-		if (_sdlog_mode == 3) {
-			add_calibration_topics();
+		if (_sdlog_profile_handle != PARAM_INVALID) {
+			param_get(_sdlog_profile_handle, &_sdlog_profile);
+		}
+
+		if (_sdlog_profile == SDLOG_PROFILE_DEFAULT) {
```

Comment:
Fixed. I meant to add that there. Also, I renamed add_default_topics to add_common_topics so that it is more clear that I'm intending to share those between estimator_replay and system_identification profiles.